hedera-mirror-importer/src/main/resources/db/migration/v1/V1.101.0_add_block_columns.sql

```diff
@@ -0,0 +1,9 @@
+alter table if exists record_file
+    add column if not exists software_version_major        int          null,
+    add column if not exists software_version_minor        int          null,
+    add column if not exists software_version_patch        int          null,
+    add column if not exists round_start                   bigint       null,
+    add column if not exists round_end                     bigint       null;
+
+alter table if exists topic_message
+alter column if exists running_hash_version drop not null;
```

Comment:
```suggestion
  alter column running_hash_version drop not null;
```

and add a new line at the end

Reply:
Also change the type to `Integer` in domain class `TopicMessage`

Reply:
Thanks I've updated it and added tests for nullable runningHashVersion.